code/modules/research/xenobiology/crossbreeding/chilling.dm

```diff
@@ -149,6 +149,9 @@ Chilling extracts:
 /obj/item/slimecross/chilling/bluespace/afterattack(atom/target, mob/user, proximity)
 	if(!proximity || !isliving(target) || active)
 		return
+	if(ismegafauna(target))
```

Comment:
might wanna just check for the TRAIT_NO_TELEPORT trait on the creature being linked instead of specifically for it being a megafauna, in case there are other things that shouldn't be moved that this extract can be used to teleport.

Reply:
Why not just use `/proc/do_teleport()`? I think it has that filter baked into it.